bash_it.sh

```diff
@@ -109,8 +108,7 @@ done
 # Custom
 BASH_IT_LOG_PREFIX="core: main: "
 _log_debug "Loading general custom files..."
-CUSTOM="${BASH_IT_CUSTOM:=${BASH_IT}/custom}/*.bash ${BASH_IT_CUSTOM:=${BASH_IT}/custom}/**/*.bash"
-for _bash_it_config_file in $CUSTOM; do
+for _bash_it_config_file in "${BASH_IT_CUSTOM:=${BASH_IT}/custom}"/*.bash "${BASH_IT_CUSTOM:=${BASH_IT}/custom}"/**/*.bash; do
```

Comment:
Using the $CUSTOM variable as it was causes the problem that this patch fixes, but an alternative might work. I'll try it out. I've alsö found the same problem when enabling a plugin, so I'm working on that too.

Reply:
I've just added a slight improvement to this, but I don't think it can be made better without reintroducing the original problem. We can't assign a glob to a variable...unless we adopt the wild compgen-sort from reloader.bash...which I alsö just fixed...

If you want to do that, then I'd say it needs to be a helper function and should be on a separate PR. This one is just to fix the quoting.